static/docs/get-started/example-pipeline.md

```diff
@@ -224,7 +225,8 @@ $ dvc run -d code/train_model.py -d data/matrix-train.pkl \
           python code/train_model.py data/matrix-train.pkl 20170426 data/model.pkl
 ```
 
-Finally, evaluate the model on the test dataset and get the metrics file:
+Finally, evaluate the model on the test dataset and get the
+[metric](/doc/commands-reference/metrics) file:
```

Comment:
I might make sense to do `abbr` for metrics term instead?

Reply:
Agreed, already requested in #461 (listed in #424).